integration/cloudbuild/testbase/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # This Dockerfile builds a base image for Trillan integration tests.
-FROM ubuntu
+FROM golang:buster
```

Comment:
What does "buster" mean? Is it based on ubuntu too?

Reply:
Buster is the latest stable Debian release

Reply:
Does it change often? Given the idea of "testbase" image being cacheable, should this be a more static version?

Reply:
Although it seems like the OS version is outside of these labels' control. We can only change Go version.